docs/DeveloperGuide.adoc

```diff
@@ -313,7 +334,55 @@ _{More to be added}_
 [appendix]
 == Use Cases
 
-(For all use cases below, the *System* is the `AddressBook` and the *Actor* is the `user`, unless specified otherwise)
+(For all use cases below, the *System* is the `TeaPet` and the *Actor* is the `Teacher`, unless specified otherwise)
+
+[discrete]
+=== Use case: Add student
+
+*MSS*
+
+1.  User keys in a student
+2.  TeaPet adds the student to the class list
++
+Use case ends.
+
+*Extensions*
+
+[none]
+* 2a. Student is invalid.
++
+[none]
+** 2a1. TeaPet shows an error message.
++
+Use case ends.
+
+[discrete]
+=== Use case: Edit student
+
+*MSS*
+
+1.  User specifies which student and what particulars he/she wants to edit
+2.  TeaPet edits the student's particulars in the class list
++
+Use case ends.
+
+*Extensions*
+
+[none]
+* 2a. Student is invalid.
++
+[none]
+** 2a1. TeaPet shows an error message.
++
+Use case ends.
++
+[none]
+* 2a. Particulars is invalid.
++
+[none]
+** 2a1. TeaPet shows an error message.
++
+Use case ends.
 
 [discrete]
 === Use case: Delete student
```

Comment:
Nice use of use cases, perhaps some of them can combine together to form a more comprehensive and detailed use case. Also, might want to revisit your Delete student use case, looks like some legacy terms still in use such as 
`AddressBook shows a list of students` & `AddressBook deletes the student`